🎉 Added document for locating files in local destination on windows (#3892)
airbytehq/airbyte: docs/tutorials/locating-files-local-destination.md

# Windows - Looking outputs for local destination (csv/json)

## Overview

This tutorial will describe how to look for json and csv files in when using local destinations on Windows on a local deployment.

There is generally some confusion when using local destination as your destination in Airbyte on Windows, especially if you are running WSL2 to power Docker. There are also two folders generated at the root folder of your Docker folder which will point you in the wrong direction

## Locating where your temp folder is

> While running Airbyte's Docker image on Windows with WSL2, you can access your temp folder by going to 

Reviewer: ```suggestion
While running Airbyte's Docker image on Windows with WSL2, you can access your temp folder by doing the following: 
```

1. Open File Explorer (Or any folder where you can access the address bar)
2. Type in `\\wsl$` in the address bar
3. The folders below will be displayed
![](../.gitbook/assets/windows-wsl2-docker-folders.png)
4. You can start digging here, but it is recommended to start searching from here and just search for the folder name you used for your local files. The folder address should be similar to `\\wsl$\docker-desktop\tmp\docker-desktop-root\containers\services\docker\rootfs\tmp\airbyte_local`

> 5. You should be able to local your local destination CSV or JSON files in this folder.

Reviewer 2: ```suggestion
5. You should be able to locate your local destination CSV or JSON files in this folder.
```

## Notes

> 1. Local JSON and Local CSV files do not persist between docker restarts. This means once you turn off your docker image, your data is lost. This is consistent with the `tmp` nature of the folder

Reviewer: ```suggestion
1. Local JSON and Local CSV files do not persist between Docker restarts. This means that once you turn off your Docker image, your data is lost. This is consistent with the `tmp` nature of the folder.
```
